lib/ansible/modules/find.py

```diff
@@ -351,15 +358,24 @@ def contentfilter(fsname, pattern, read_whole_file=False):
 
     prog = re.compile(pattern)
 
+    file_args = dict(file=fsname)
+    if PY3:
+        file_args['encoding'] = encoding
```

Comment:
2.17 requires Python 3.7 or later for module execution, and I think you can remove the PY3 check.